src/bp/ml/svm/index.ts

```diff
@@ -76,6 +75,11 @@ export class Trainer implements sdk.MLToolkit.SVM.Trainer {
   isTrained(): boolean {
     return !!this.model
   }
+
+  private _extractSeed(options?: sdk.MLToolkit.SVM.SVMOptions): number {
+    const seed = options?.seed
+    return seed ?? Math.round(Math.random() * 10000)
```

Comment:
I thought that was handled by libsvm itself . I imagine it makes no difference

Reply:
yes, it already handled in the binding, but I wanted to make the seed mandatory.